src/app/routes/_profile.router.js

```diff
@@ -190,8 +190,9 @@ export default (app, router) => {
    */
   router.put('/api/profile', async (req, res) => {
     // remove readonly data
-    if (req.body.local) delete req.body.local;
+    if (req.body.local.password) delete req.body.local.password;
```

Comment:
No this will just remove `password` from the post request. The password will be generated with `newPassword`: https://github.com/Sirimangalo/meditation-plus-server/blob/b5cb801a655788681f80aa54d9c2979117bd1c50/src/app/routes/_profile.router.js#L202